server/routes/api/user.js

```diff
@@ -21,7 +109,19 @@ router.get('/profile', function(req, res) {
                     email_verified: user.email_verified,
                     full_name: user.full_name,
                     birthday: user.birthday,
-                    groups: groups
+                    groups: groups,
```

Comment:
can you pass back something that says whether the resume is there too, best for user is probably the original filename but then we have to preserve that, so maybe just a boolean would be fine